docker-compose.yml

```
@@ -205,6 +206,42 @@ services:
     networks:
       - net
 
+  rabbitmq:
+    image: rabbitmq:4.0
+    hostname: rabbitmq
+    ports:
+      - "5672:5672"
+    networks:
+      - net
```

The mail container restarts if rabbitmq is not ready yet. I'd like to suggest to use a healthcheck that can be used as a condition in the mail service:

```suggestion
      - net
    healthcheck:
      test: ["CMD", "rabbitmq-diagnostics", "-q", "ping"]
      interval: 1m30s
      timeout: 30s
      retries: 5
      start_period: 30s
```